_component_design/buttons.md

```diff
@@ -233,32 +190,20 @@ Compact buttons should be used when you have severe space constraints, such as i
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Button sets
+## Primary button
 
-- Buttons should be aligned in order of primary, secondary, and tertiary.
-- Button sets are most often used in Popovers, Modals or Create/Edit Forms to group the submit, back or next, and cancel buttons.
-- A button set should never contain more than one primary button.
-- Avoid using split buttons in a button set as it would complicate what should be a simple decision.
-- The cancel action must always be an `hxBtn.hxLink` as shown in the example.
+A primary button indicates the primary action a user can take and is designed to command the attention of users more than any other content on a page.
 
-When submitting forms:
+Do not use multiple primary buttons in the same view. Multiple primary buttons can cause visual confusion and increase cognitive load while decreasing the efficacy of the button to direct user attention to a single action.
 
-- Disable all buttons, and change the primary button to loading state.
+For example, do not use primary buttons in a dashboard widget because it attracts too much attention and the user cannot settle and focus on a single widget.
```

Comment:
should probably add the content about button states:
"A primary button can exist in one of the following states: default, hover, pressed, or disabled."